files/usr/bin/cinnamon-settings

```diff
@@ -120,9 +121,9 @@ if len(sys.argv) > 1:
             os.execvp("/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py", (" ",) + tuple(sys.argv[1:]))
     elif os.path.exists("/usr/share/cinnamon/cinnamon-settings/modules/cs_%s.py" % module):
         os.execvp("/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py", (" ",) + tuple(sys.argv[1:]))
-    elif os.path.exists("/usr/bin/cinnamon-control-center"):
-        os.execvp("/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py", (" ",) + tuple(sys.argv[1:]))
-    elif os.path.exists("/usr/bin/gnome-control-center"):
+    elif shutil.which("cinnamon-control-center"):
+        os.execvp("cinnamon-control-center", (" ",) + tuple(sys.argv[1:]))
```

Comment:
This still needs to call cinnamon-settings.py - cinnamon-control-center is only used for debugging purposes mostly. 

This particular check doesn't seem all that useful in the first place - we have some C modules that are part of cinnamon-control-center, but we load them via cinnamon-settings.  We really should be looking at names of those modules themselves (in `...lib/.../cinnamon-control-center-1/panels`) rather than just the binary. But it's not a big deal in reality.

Reply:
Done, thanks for the heads up!